llvm/lib/Target/SBF/MCTargetDesc/SBFMCCodeEmitter.cpp

```diff
@@ -93,17 +93,19 @@ unsigned SBFMCCodeEmitter::getMachineOpValue(const MCInst &MI,
   if (MI.getOpcode() == SBF::JAL)
     // func call name
     Fixups.push_back(MCFixup::create(0, Expr, FK_PCRel_4));
-  else if (MI.getOpcode() == SBF::LD_imm64)
+  else if (MI.getOpcode() == SBF::LD_imm64 ||
+           MI.getOpcode() == SBF::MOV_32_64_addr)
     Fixups.push_back(MCFixup::create(0, Expr, FK_SecRel_8));
-  else
+  else if (MI.getOpcode() == SBF::HOR_addr) {
+    // This is already handled in MOV_32_64
+  } else
```

Comment:
Please remove the above no-op if-test. Then add a comment in the code handling `MOV_32_64` why `HOR_addr` does not need explicit handling.